packages/ua-devtools/src/oapp/config.ts

```diff
@@ -92,26 +92,46 @@ export const configureOAppPeers: OAppConfigurator = withOAppLogger(
     }
 )
 
-export const configureSendLibraries: OAppConfigurator = async (graph, createSdk) =>
-    flattenTransactions(
-        await Promise.all(
-            graph.connections.map(async ({ vector: { from, to }, config }): Promise<OmniTransaction[]> => {
+export const configureSendLibraries: OAppConfigurator = withOAppLogger(
+    createConfigureEdges(
+        withOAppLogger(
+            async ({ vector: { from, to }, config }, sdk): Promise<OmniTransaction[]> => {
+                const logger = createOAppLogger()
+
                 if (!config?.sendLibrary) {
+                    logger.verbose(`sendLibrary configuration not set for ${formatOmniVector({ from, to })}, skipping`)
                     return []
                 }
 
-                const oappSdk = await createSdk(from)
-                const endpointSdk = await oappSdk.getEndpointSDK()
+                const endpointSdk = await sdk.getEndpointSDK()
                 const isDefaultLibrary = await endpointSdk.isDefaultSendLibrary(from.address, to.eid)
                 const currentSendLibrary = await endpointSdk.getSendLibrary(from.address, to.eid)
 
                 if (!isDefaultLibrary && currentSendLibrary === config.sendLibrary) {
+                    logger.verbose(
+                        `Current sendLibrary is not default library and is already set to config.sendLibrary for ${formatOmniVector({ from, to })}, skipping`
```

Comment:
will replace `config.sendLibrary` with `${config.sendLibrary}` and also specify it in the subsequent log when it is being set